packages/core/src/metrics/aggregator.ts

```diff
@@ -62,7 +63,11 @@ export class MetricsAggregator implements MetricsAggregatorBase {
     const tags = sanitizeTags(unsanitizedTags);
 
     const bucketKey = getBucketKey(metricType, name, unit, tags);
```

Comment:
I wonder if we should type bucket key stronger with template literal types - we can do it in a follow up PR if you think it's valuable.